src/slidedown/Slidedown.ts

```diff
@@ -14,7 +14,6 @@ import { SlidedownPermissionMessageOptions } from '../models/Prompts';
 import { SERVER_CONFIG_DEFAULTS_SLIDEDOWN } from '../config';
 import getLoadingIndicatorWithColor from './LoadingIndicator';
 import getDialogHTML from './DialogHTML';
-import sanitizeHtml from 'sanitize-html';
```

Comment:
After discussion with Josh, we figured this should most likely be done backend-side or on dashboard